constants/metalContent.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * Represent a feature for the metal page
+ */
+interface MetalFeature {
+  /** The visible title of the feature */
+  title: string
+  /** The visible description of the feature */
+  description: string
+  /** The image of the feature */
+  image: string
+}
+
+const METAL_FEATURES: Array<MetalFeature> = [
+  {
+    title: 'Automation',
+    description: 'Metal is designed with the vision to seamlessly automate classical and quantum analysis. Choose from a list of available tools or connect your own through a flexible plug-in interface that allows you to orchestrate device design from within Metal. When you are ready, with the click of a button, export the design for fabrication, and you are done.',
+    image: '/images/metal/features/automation.png'
+  },
+  {
+    title: 'Flexible & extensible',
+    description: `<strong>Flexible interface:</strong> GUI view, Jupyter notebook view, Simulator view, Physical output view.<br><br>
```

Comment:
Let's use paragraphs instead of `<br>` for separating the content, you can always style with non-scoped CSS styles.